src/js/sign.js

```diff
@@ -0,0 +1,221 @@
+const emailDiv = document.querySelector('.content1_div2_div1'); //이메일 상위 div
+const pwDiv = document.querySelector('.content1_div2_div2_div-input'); //비밀번호 상위 div
+const emailInput = document.querySelector('.content1_div2_div1_input'); //이메일 input
+const pwInput = document.querySelector('.content1_div2_div2_input'); //비밀번호 input
+const pwSignupDiv1 = document.querySelector('.signup_pw1'); //signup 비밀번호 상위 div1
```

Comment:
주석처리가 꼼꼼히 되어있어서 보기 좋았어요!
그런데 주석보면 이해가 되지만 주석이 없다면 뭘 뜻하는 지 헷갈릴 것 같아요 🤔

```suggestion
const pwInput = document.querySelector('.pw_input');
const pwSignupWrapper = document.querySelector('.signup_pw_wrapper');
```

개인 스타일이지만 저였으면 이렇게 했을 것 같다는 거 **조용히** 공유해드려요!

Reply:
저도 주석이 없으면 코드 짜다가 이게 어떤 변수였나 까먹을 때가 있어서 class명들을 다시 한번 바꾸긴 해야될 거 같다고 느꼈습니다!

Reply:
오 Suggested... 👍